src/components/PopoverWithMeasuredContent.tsx

```diff
@@ -1,56 +1,21 @@
-import PropTypes from 'prop-types';
+import isEqual from 'lodash/isEqual';
 import React, {useMemo, useState} from 'react';
-import {View} from 'react-native';
-import _ from 'underscore';
+import {LayoutChangeEvent, View} from 'react-native';
 import useThemeStyles from '@hooks/useThemeStyles';
 import useWindowDimensions from '@hooks/useWindowDimensions';
 import PopoverWithMeasuredContentUtils from '@libs/PopoverWithMeasuredContentUtils';
 import CONST from '@src/CONST';
 import Popover from './Popover';
-import {defaultProps as defaultPopoverProps, propTypes as popoverPropTypes} from './Popover/popoverPropTypes';
-import {windowDimensionsPropTypes} from './withWindowDimensions';
+import {PopoverProps} from './Popover/types';
 
-const propTypes = {
-    // All popover props except:
-    // 1) anchorPosition (which is overridden for this component)
-    // 2) windowDimensionsPropTypes - we exclude them because we use useWindowDimensions hook instead
-    ..._.omit(popoverPropTypes, ['anchorPosition', ..._.keys(windowDimensionsPropTypes)]),
-
-    /** The horizontal and vertical anchors points for the popover */
-    anchorPosition: PropTypes.shape({
-        horizontal: PropTypes.number.isRequired,
-        vertical: PropTypes.number.isRequired,
-    }).isRequired,
-
-    /** How the popover should be aligned. The value you passed will is the part of the component that will be aligned to the
-     * anchorPosition. ie: vertical:top means the top of the menu will be positioned in the anchorPosition */
-    anchorAlignment: PropTypes.shape({
-        horizontal: PropTypes.oneOf(_.values(CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL)),
-        vertical: PropTypes.oneOf(_.values(CONST.MODAL.ANCHOR_ORIGIN_VERTICAL)),
-    }),
-
-    /** Static dimensions for the popover.
-     * Note: When passed, it will skip dimensions measuring of the popover, and provided dimensions will be used to calculate the anchor position.
-     */
-    popoverDimensions: PropTypes.shape({
-        height: PropTypes.number,
-        width: PropTypes.number,
-    }),
+type AnchorPosition = {
```

Comment:
We define this type in 3 different files ([styles/index.ts](https://github.com/Expensify/App/blob/main/src/styles/index.ts#L47), [libs/calculateAnchorPosition.ts](https://github.com/Expensify/App/blob/main/src/libs/calculateAnchorPosition.ts#L12) and `PopoverWithMeasuredContent`)
Let's move it to `src/components/Popover/types.ts` and export it to those 3 files

Reply:
type reusing added